egs/tedlium3/ASR/transducer_stateless/pretrained.py

```diff
@@ -0,0 +1 @@
+../../../librispeech/ASR/transducer_stateless/pretrained.py
```

Comment:
You have made a change in the decoder to add `unk_id`, so `pretrained.py` cannot be reused, I think. Please have a test for your `pretrained.py` and make sure it works.